tutorials/smart-contracts/deploy-nft.md

```diff
@@ -0,0 +1,120 @@
+
+## Creating the NFT Contract
+
+To create the NFT contract, you can follow the steps below:
+
+1. Navigate to the [Polkadot Remix IDE](https://polkadot.remix.io){target=\_blank}
+2. Click in the **Create new file** button under the **contracts** folder, and name your contract as `MyNFT.sol`
+
+    ![](/images/tutorials/smart-contracts/deploy-nft/deploy-nft-1.webp)
+
+3. Now, paste the following NFT contract code into the editor
+
+    ```solidity title="MyNFT.sol"
+    --8<-- 'code/tutorials/smart-contracts/deploy-nft/MyNFT.sol'
+    ```
+
+    The key components of the code above are:
+
+    - Contract Imports
+
+        - [`ERC721.sol`](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v5.0.0/contracts/token/ERC721/ERC721.sol){target=\_blank} - the base contract for non-fungible tokens, implementing core NFT functionality like transfers and approvals
+        - [`Ownable.sol`](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v5.0.0/contracts/access/Ownable.sol){target=\_blank} - provides basic authorization control, ensuring only the contract owner can mint new tokens
+    
+    - Constructor Parameters
+
+        - `initialOwner` - sets the address that will have administrative rights over the contract
+        - `"MyToken"` - the full name of your NFT collection
+        - `"MTK"`- the symbol representing your token in wallets and marketplaces
+
+    - Key Function
+
+        - [`_safeMint(to, tokenId)`](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v5.0.0/contracts/token/ERC721/ERC721.sol#L304){target=\_blank} - an internal function from `ERC721` that safely mints new tokens. It includes checks to ensure the recipient can handle ERC721 tokens (important when minting to smart contracts)
+
+    !!! tip
+        Use the [OpenZeppelin Contracts Wizard](https://wizard.openzeppelin.com/){target=\_blank} to generate customized smart contracts quickly. Simply configure your contract, copy the generated code, and paste it into Polkadot Remix IDE for deployment.
+
+## Compiling the Contract
+
+1. Select the **Solidity Compiler** plugin from the left panel
+
+    ![](/images/tutorials/smart-contracts/deploy-nft/deploy-nft-3.webp)
+
+3. Click in the **Compile MyNFT.sol** button
+
+    ![](/images/tutorials/smart-contracts/deploy-nft/deploy-nft-4.webp)
+
+4. If the compilation succeeded, you can see a green checkmark indicating success in the **Solidity Compiler** icon
+
+    ![](/images/tutorials/smart-contracts/deploy-nft/deploy-nft-5.webp)
+
+## Deploying the Contract
+
+1. Select the **Deploy & Run Transactions** plugin from the left panel
+
+    ![](/images/tutorials/smart-contracts/deploy-nft/deploy-nft-6.webp)
+
+2. Configure the deployment settings
+    1. From the **ENVIRONMENT** dropdown, select **Westend Testnet - MetaMask**
+    2. From the **ACCOUNT** dropdown, select the account you want to use for the deploy
+
+        ![](/images/tutorials/smart-contracts/deploy-nft/deploy-nft-7.webp)
```

Comment:
All images that contain steps should be indented as the parent list item. Check other ocurrences
```suggestion
    ![](/images/tutorials/smart-contracts/deploy-nft/deploy-nft-7.webp)
```